tests/test_custodia.py

```diff
@@ -32,7 +32,7 @@ def find_port(host='localhost'):
 [global]
 server_version = "Secret/0.0.7"
 server_url = ${SOCKET_URL}
-auditlog = test_audit.log
+auditlog = tests/tmp/test_audit.log
```

Comment:
SOCKET_URL is required for different sockets (TCP/IP, Unix stream). The test directory is always the same. It's fine to hard code it for tests.